xonsh/parsers/fstring_adaptor.py

```diff
@@ -0,0 +1,182 @@
+import re
+from xonsh.lazyasd import lazyobject
+from ast import parse as pyparse
+from xonsh import ast
+from xonsh.platform import PYTHON_VERSION_INFO
```

Comment:
Please reorder these imports to be in the following groups, as per PEP8:

1. standard library imports
2. xonsh imports

Reply:
Done.